src/style.css

```diff
@@ -34,16 +34,12 @@ header {
   color: purple;
 }
 
-.display {
+/* .display {
```

Comment:
I think we can remove these unused stylings if we're replacing them with MUI components used in this PR.